tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "allowJs": false,
     "checkJs": false,
     "noEmit": true,
-    "types": ["svelte", "jest", "cypress"],
+    "types": ["svelte", "jest", "cypress", "node"],
```

Comment:
@pooja169usp why did we need node types added? 🤔

Reply:
Oh this should not be in there, I added it because I thought we were being returned binary file and needed Buffer. But the file is returned as base64, so we don't need this any more, will remove it.